.github/CODEOWNERS

```diff
@@ -6,8 +6,14 @@
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-/**                      @rickwinter @LarryOsterman
-/sdk/                    @rickwinter @LarryOsterman
+# /**
```

Comment:
Why did we comments this out? And replace it with `/*`?

Reply:
Following the pattern of every other language SDK. We can add it back if we want to, but I don't see reason too given its now aligned with the teams other repos.

Reply:
We need to add this back.

Without it, we have unowned paths with no owners, such as `.vscode/cspell.json`
For example, see this, which doesn't list any codeowner approvers:
https://github.com/Azure/azure-sdk-for-cpp/pull/6198

Look at Java:
https://github.com/Azure/azure-sdk-for-java/blob/0f17a67a010544bb63b67440f20775b9fffa59e2/.github/CODEOWNERS#L18

Otherwise, like in .NET, these paths become "orphaned":
https://github.com/Azure/azure-sdk-for-net/blob/22109d25974000bd523ca3ccec960edc14fbddc5/.github/CODEOWNERS#L16-L29

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/6198